graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH7.kt

```diff
@@ -182,9 +102,9 @@ class BEH7 : GraphScenario() {
     }
 
     @Eac(""""
-        |When a directed graph has all edges declared in both directions
+        When a given graph has all edges declared in both directions
         and detecting only simple cycles, return 5
```

Comment:
🚫 Besides the running comments, what does it mean that we are "detecting only simple cycles"? The function doesn't seem to have any alternative behaviour! If there is a restriction on the behaviour, then it is even more important that we define exactly what is being counted.

More to the point, though, as written this is just a unit test. If there is an underlying AC, you should identify it and write that instead (you can keep the example scenario, of course).

Reply:
⛏️ Also, formatting is way off.